### ISSUE TITLE
[BUG] fix `releases.yml` workflow version number

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/SocialSlash#16]
### Description

Update the 2 actions to the latest version

```diff
- uses: actions/checkout@v2
+ uses: actions/checkout@v3
```

```diff
- uses: TriPSs/conventional-changelog-action@v3.7.1
+ uses: TriPSs/conventional-changelog-action@v3
```

File path - https://github.com/Pradumnasaraf/SocialSlash/blob/main/.github/workflows/releases.yml

**Comment by tarunsamanta2k20** [comment]
I would like to work on this issue

**Comment by gauravrattan** [comment]
Pls assign @Pradumnasaraf

**Comment by Pradumnasaraf** [comment]
> Pls assign @Pradumnasaraf

Hey, I have already assigned you one issue on a different repo. Please let other contributors take part in it.

**Comment by dixitrathi7** [comment]
Pls assign me @Pradumnasaraf ,thanks

**Comment by Pradumnasaraf** [comment]
> Pls assign me @Pradumnasaraf ,thanks

Go ahead

**Comment by dixitrathi7** [comment]
pls check @Pradumnasaraf  I will update, thanks

**Comment by Pradumnasaraf** [comment]
fixed in #25
